src/app/team/recruit/[projectId]/page.tsx

```diff
@@ -0,0 +1,12 @@
+import { TeamRecruitForm } from '@/components/domain/team/recruit/TeamRecruitForm';
+
+interface TeamRecruitPageProps {
+  params: {
+    projectId: string;
+  };
+}
+
+// NOTE: 추후 API 연동 시 권한 체크 로직 추가
+export default function TeamRecruitPage({ params: { projectId } }: TeamRecruitPageProps) {
+  return <TeamRecruitForm projectId={projectId} />;
```

Comment:
```suggestion
return <TeamRecruitForm projectId={Number(projectId)} />;
```

projectId가 다른 곳들에서 모두 number로 관리되고 있어서, 여기서 정수로 변환하고 number 타입으로 관리하는게 좋을 듯 합니다!

Reply:
수정했습니다! ✅ [a276319](https://github.com/PRism-Works/PRism-FE/pull/194/commits/a276319a0d882ef47fca66b68ddd824abaccb01f)